test/service/test_async_manifest_service.py

```diff
@@ -131,7 +131,7 @@ def run(self, result: Optional[unittest.result.TestResult] = ...) -> Optional[un
         with mock.patch('azul.service.manifest_service.ManifestService.__init__') as init_:
             init_.return_value = None
             with mock.patch('azul.service.manifest_service.ManifestService.get_cached_manifest') as get_cache_manifest:
```

Comment:
```suggestion
            with mock.patch('azul.service.manifest_service.ManifestService.get_cached_manifest') as get_cached_manifest:
```

Reply:
I shortened the variable, so it's partially addressed. :)